wpeverest/everest-forms
# Dev- Calculation Module

### All Submissions:






### Changes proposed in this Pull Request:



Priviously, no complex calculation like if, else, sqrt, floor, etc. This PR add complex calculation.

### How to test the changes in this Pull Request:

1. Create new form and Drag Calculation field ( number and single item )
2. Enable calculation and insert formula.
3.  Save form and precview and perform calculation.
4. verify calculated value is corect or not

Note: Calculation supported field : Number, Single Item for caculation result. 
'text', 'select', 'radio', 'checkbox', 'number', 'name', 'first-name', 'lsat-name', 'email', 'url', 'payment-checkbox', 'payment-multiple', 'payment-single', 'payment-total'

Logical Operator support : '&&', '||', '/', '==', '>', '>=', '-', '%', '*', '!=', '+', '<', '<=', '!', '-1'

Dependency PR:
[PR](https://github.com/wpeverest/everest-forms-pro/pull/935)
### Types of changes:




### Other information:




### Changelog entry

> Dev- Calculation Module
